.eslintrc.json

```diff
@@ -0,0 +1,52 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "node": true,
+      "jest": true
+    },
+    "extends": [
+      "airbnb"
+    ],
+    "globals": {
+      "Atomics": "readonly",
+      "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+      "ecmaFeatures": {
+        "jsx": true
+      },
+      "ecmaVersion": 2015 ,
+      "sourceType": "module"
+    },
+    "plugins": [
+      "react",
+      "prettier"
+    ],
+    "rules": {
+      "import/prefer-default-export": "off",
+
+      "react/jsx-one-expression-per-line": [
+        0,
+        {
+          "allow": "none"
+        }
+      ],
+      "react/jsx-props-no-spreading": [
+        0,
+        {
+          "html": "ignore",
+          "custom": "ignore"
+        }
+      ],
+      "no-underscore-dangle": [
+        2,
+        {
+          "allow": [
+            "_id"
+          ]
+        }
+      ]
+    }
+  }
+  
```

Comment:
Can we have a newline here as well? Thanks!

Reply:
@ahhda The request changes has been performed, kindly approve.